<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfParas</class>
 <widget class="QWidget" name="ConfParas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	   height: 18px;
       background-color:#76becc;
       border-radius: 5px; 
       border: 2px groove gray;
       border-style: outset;
}

QPushButton::hover{
       background-color:#228fbd;
       border-radius: 5px; 
       border: 1px groove gray;
       border-style: outset;
}

QPushButton::pressed{
       background-color:#009ad6;
       border-radius: 5px; 
       border: 1px groove gray;
       border-style: outset;
}

QLineEdit{
border-radius: 7px;
height: 20px;
background-color:#f2eada;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>306</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ClearButton">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RunButton">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>Start a Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar#progressBar{
      border:none;
      background:#d3d7d4;
      border-radius:5px;
      text-align:center;
      height:15px;
}
 
QProgressBar::chunk {
 
      background-color:#7fb80e;
      border-radius:4px;
      
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="LayersParamsLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Refractive Index/Absorption Coefficient/Scattering Coefficient/Anisotropy/Thickness of each layer.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="text">
       <string>Specify Parameters For One Run</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="PNhorizontalLayout">
        <item>
         <widget class="QLabel" name="PhoNumLabel">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Number Of Photons:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="PhoNumSpinBox">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#f2eada;</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="PhoNumSlider">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="NohorizontalLayout">
      <item>
       <widget class="QLabel" name="No_zra_Label">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>No. of z/r/alpha Grid Elements:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="No_zGridEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="No_rGridEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="No_aGridEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="MediumhorizontalLayout">
      <item>
       <widget class="QLabel" name="MedLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>Refractive Index for medium Above/Below:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MedAboveEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MedBelowEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="GridhorizontalLayout">
      <item>
       <widget class="QLabel" name="zrGridLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>delta z/delta r :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zGridEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rGridEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>Number Of Layers:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="num_layer_spin">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#f2eada;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SpeLayerButton">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>Specify Layer Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PhoNumSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PhoNumSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
